src/Transport/Receiving/MessageReceiverNotifier.cs

```diff
@@ -20,9 +21,10 @@ public MessageReceiverNotifier(IManageMessageReceiverLifeCycle clientEntities, I
             this.settings = settings;
             locksTokensToComplete = new ConcurrentStack<Guid>();
             batchedCompletionCts = new CancellationTokenSource();
+            RefCount = 1;
         }
 
-        bool ShouldReceiveMessages => !stopping || isRunning;
+        bool ShouldReceiveMessages => !stopping;
```

Comment:
This is valid as once the endpoint is stopped (`isRunning == false`) it won't be restarted. The `isStopping` is not reset to `false` as well.